docs/source/pre_application/digital_ocean_tutorial.rst

```diff
@@ -0,0 +1,134 @@
+    $ export DIGITAL_OCEAN_KEY_FINGERPRINT=28:38:e7xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx:ca:5c
+
+
+Setup Remote Node
+-----------------
+Locally, we will install `NuCypher Ops <https://github.com/nucypher/nucypher-ops>`_ to handle the heavy lifting of setting up a node.
+
+.. code-block:: bash
+    $ git clone git@github.com:nucypher/nucypher-ops.git
+    $ cd nucypher-ops
+    $ pip3 install -e .
+
+Now NuCypher Ops is installed we can create a droplet on Digital Ocean:
+
+.. code-block:: bash
+    nucypher-ops nodes create --network ibex --count 1 --cloudprovider digitalocean
+
+At this point you should see the droplet in your Digital Ocean dashboard.
+Now we can deploy the PRE Node:
+
+.. code-block:: bash
+    nucypher-ops ursula deploy --eth-provider $INFURA_RINKEBY_URL --nucypher-image nucypher/nucypher:experimental --payment-provider $INFURA_MUMBAI_URL --network ibex
+
+This should produce a large amount of logs as the ansible plays install all the requirements and setup the node.
+The final output should be similar to:
+
+.. code-block:: bash
+    some relevant info:
+    config file: "/SOME_PATH/nucypher-ops/configs/ibex/nucypher/ibex-nucypher.json"
+    inventory file: /SOME_PATH/nucypher-ops/configs/ibex-nucypher-2022-03-25.ansible_inventory.yml
+    If you like, you can run the same playbook directly in ansible with the following:
+        ansible-playbook -i "/SOME_PATH/nucypher-ops/configs/ibex-nucypher-2022-03-25.ansible_inventory.yml" "src/playbooks/setup_remote_workers.yml"
+    You may wish to ssh into your running hosts:
+        ssh root@123.456.789.xxx
+    *** Local backups containing sensitive data may have been created. ***
+    Backup data can be found here: /SOME_PATH//nucypher-ops/configs/ibex/nucypher/remote_worker_backups/
+
+This tells us the location of several config files and helpfully prints the ip of our newly created node (you can also see this on the Digital Ocean dashboard).
+Let's `ssh` into it and look at the logs:
+
+.. code-block:: bash
+    $ ssh root@123.456.789.xxx
+    root@nucypher-ibex-1:~#
+    root@nucypher-ibex-1:~# sudo docker logs --follow ursula
+    ...
+    ! Operator 0x06E11400xxxxxxxxxxxxxxxxxxxxxxxxxxxx1Fc0 is not funded with ETH
+    ! Operator 0x06E11400xxxxxxxxxxxxxxxxxxxxxxxxxxxx1Fc0 is not bonded to a staking provider
+    ...
+
+These lines will print repeatedly until the Operator is funded with some ETH and bonded to a staking provider.
+Send the address that is printed some testnet eth.
+
+
+Stake and Bond
+--------------
+Now that our operator is funded with some eth, we're ready to stake and bond.
+At this point you need some testnet eth and 40K T in a metamask wallet.
+Again, ask in the discord if you need help with this.
+
+Navigate to the `Testnet Staking Dashboard <https://dn3gsazzaajb.cloudfront.net/manage/stake>`_ and connect your metamask wallet.
+Go to the `stake` tab and click "Stake liquid T on rinkeby"
+
+.. image:: ../.static/img/testnet_stake_dashboard.png
```

Comment:
The Button image for staking liquid T on Threshold has the nucypher logo in it. Should be the T logo -> T logo

Reply:
^ @vepkenez